schema.sql

```diff
@@ -72,3 +72,11 @@ CREATE TABLE visits (
     FOREIGN KEY (vets_id)
     REFERENCES public.vets (id)
 )
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+ALTER TABLE owners ALTER COLUMN age DROP NOT NULL;
+
+CREATE INDEX idx_animals_id ON visits(animals_id);
```

Comment:
👏 Great job with the initial analysis and the inclusion of screenshots. However, there's a specific step to follow as per the project requirements:

- [ ] Before creating the index, the execution time is 603ms, which is less than 1s. As stated in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depending%20on,bigger%20than%201000ms.), "Check that by running explain analyze query: - If you get Execution time: X ms and X >= 1000: that should be enough, and you can continue to the project requirements. - If you get Execution time: X ms and X < 1000: please go back to point 3. and repeat until you get a value bigger than 1000ms." 
- [ ] Please remember to run the query again and update your screenshots. 🙏 

To meet this requirement, kindly follow the steps mentioned by the previous reviewer and update your screenshot accordingly. This will ensure that your analysis aligns perfectly with the project specifications. Keep up the good work! 👍📊🕐

Reply:
@shahadat3669 Hello,
Thank you for your review! 
I fixed all issues except animals_id with seq scan (in the picture it shows less than 1000ms). To fix this issue I tried disabling indexing with the command ALTER INDEX idx_animals_id DISABLE; it didn't work. I re-run preparation step 3, however I still see the indexing. Can you help me to un-index idx_animals_id, please?